### DISCUSSION TITLE
TeamTalk v5.10

### DISCUSSION THREAD

**Discussion by bear101** [BearWare/TeamTalk5#1402] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/BearWare/TeamTalk5/releases/tag/v5.10'>TeamTalk v5.10</a>.</em>

**Comment by ahmednaalik51** [comment]
Hello,
This version great. But I have question. Will you stay with TeamTalk 5 or will it eventual go to TeamTalk 6?
If it will go to TeamTalk 6 then I have idea. Because so many people now are use TeamTalk 5.
Is there posible way to make so that TeamTalk 5 clients could connect to TeamTalk 6 servers?
Because so many people are use now TeamTalk 5 and migrating everyone will be very bad for some, because lots of people don't moved around right away.
Also it could be like other services such as Skype or others that allow older clients to still work for a time.
I am just wondering if it is possible to make them compatibile and when you wanted to start next on TeamTalk 6.
Many people are say also that instead of calling it TeamTalk 5 or TeamTalk 6 that you should just call it TeamTalk and just have new versions expire after a time. What did you think about that?

**Comment by lbk2907** [comment]
What I know, the next version is TeamTalk 5.11.

**Comment by computertech1012** [comment]
Hi,
Is anyone else having problems with the bearware web login features on TeamTalk 5.10?
If I try to login with my bearware user account, I get an error that it says faild to authenticate, even though my username and password is typed correctly.
I am running windows 10, and everything is also allowed in the firewall. How can I fix this problem?
I don't know if it effects users that already have bearware accounts signed in or not, but if you try to activate bearware, it always says that it faild.
@bear101 How can I solve?
Also @lbk2907 Any ideas?

**Comment by bear101** [comment]
What is your web-login name? Also can you sign in to the account in the TeamTalk client?

**Reply by computertech1012** [reply]
I just type in my email address and password on the web, and it works well.
But I am unable to sign in at all in TeamTalk, I get an error message right away saying it can't authenticate, and it doesn't even work, but on 5.9.1, for example, it works fine. So maybe when installing TeamTalk the files that you said had to be deploied didn't get deploid to some people? I don't know.
I wish TeamTalk had a log feature, so we can post logs of the event.

**Reply by bear101** [reply]
I can reproduce the problem: https://github.com/BearWare/TeamTalk5/issues/1425
I'll try to fix it.

**Reply by bear101** [reply]
If you download TeamTalk Pro artifacts from this build then the issue should be fixed. Note, however, that you have to reauthenticate.
https://github.com/BearWare/TeamTalk5/actions/runs/2827347238

**Reply by bear101** [reply]
I had to make a correction :(
You can download artifacts for each platform here: https://github.com/BearWare/TeamTalk5/actions?query=branch%3Aissue-1425

**Comment by lbk2907** [comment]
First, Do you have correctly install Microsoft Visual C++? Second, are you
have try on the portable version?
